### PR TITLE
Upgrade loaders.gl and fix image loading

### PR DESCRIPTION
**Pull request by xintongxia**
#### Background

for #2976

`loadImage` is moved from `loaders.gl/core`, now in its own package `@loaders.gl/images`

#### Change List
- bump `loaders.gl/*` to `alpha.4`
- fix `loadIamge` from @loaders.gl/images`

### CODE REVIEW

File: examples/layer-browser/package.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
     "start-local-production": "webpack-dev-server --env.local --env.production --progress --hot --open"
   },
   "dependencies": {
-    "@loaders.gl/core": ">=1.0.0-alpha.2",
-    "@loaders.gl/ply": ">=1.0.0-alpha.2",
+    "@loaders.gl/core": "^1.0.0-alpha.4",
+    "@loaders.gl/ply": "^1.0.0-alpha.4",
     "@luma.gl/addons": "^7.0.0-beta.4",
```
Comment (ibgreen) [inline examples/layer-browser/package.json]: Update 
- loaders to `^1.0.0` 
- luma to `7.0.0-rc.1`!

File: modules/layers/package.json [hunk 2]
```diff
@@ -29,7 +29,8 @@
     "prepublishOnly": "npm run build-bundle && npm run build-bundle -- --env.dev"
   },
   "dependencies": {
-    "@loaders.gl/core": "^0.6.2",
+    "@loaders.gl/core": "^1.0.0-alpha.4",
```
Comment (ibgreen) [inline modules/layers/package.json]: Ditto

File: docs/layers/scenegraph-layer.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ The `ScenegraphLayer` renders a number of instances of a complete luma.gl sceneg
 ```js
 import DeckGL from '@deck.gl/react';
 import {ScenegraphLayer} from '@deck.gl/mesh-layers';
-import {loadFile, registerLoaders} from '@loaders.gl/core';
```
Comment (ibgreen) [inline docs/layers/scenegraph-layer.md]: loadFile wasn't used?

Comment (xintongxia) [reply docs/layers/scenegraph-layer.md]: no used

File: examples/layer-browser/package.json (hunk 1 above)
Comment (xintongxia) [reply examples/layer-browser/package.json]: I am thinking to make an script to upgrade these.... We have more and more modules now

**Comment by Pessimistress** [comment]
Isn't `loadFile` deprecated? (Used in ScenegraphLayer and SimpleMeshLayer)

File: examples/layer-browser/package.json (hunk 1 above)
Comment (Pessimistress) [reply examples/layer-browser/package.json]: Theoretically we won't have to upgrade these again until loaders.gl v2.

**Pull request merged**